packages/neutrino/src/api.js

```diff
@@ -26,14 +27,28 @@ const pathOptions = [
 
 // getOptions :: Object? -> IO Object
 const getOptions = (opts = {}) => {
+  let fileExtensions = new Set(['js', 'jsx', 'vue', 'ts', 'mjs']);
   const options = merge({
     env: {
       NODE_ENV: 'development'
     },
     debug: false,
-    quiet: false
+    quiet: false,
+    extensions: fileExtensions
   }, opts);
 
+  Object.defineProperty(options, 'extensions', {
+    enumerable: true,
+    get() {
+      return [...fileExtensions];
+    },
+    set(extensions) {
+      const newExtensions = extensions.map(ext => equals(ext[0], '.') ? tail(ext) : ext);
+
+      fileExtensions = [...new Set(newExtensions)];
```

Comment:
Looks like the original value of `fileExtensions` is a Set, but here we spread it into an array. We can keep this as a Set here, and only spread to array during `get`:

```js
fileExtensions = new Set(newExtensions);
```

Reply:
Combining this with my previous comment, this can be one-lined into:

```js
fileExtensions = new Set(extensions.map(ext => ext.replace('.', '')));
```